Cargo.toml

```diff
@@ -73,6 +73,9 @@ members = [
     "tesseract/fees/prisma-cli",
     "tesseract/telemetry",
     "tesseract/config",
+
+    # integration tests
+    "parachain/integration-test",
```

Comment:
Should be in the tesseract folder no?

Reply:
Not really if you check the structure of integration tests are related and inside the module directory

Reply:
But these are tesseract integration tests no?